### ISSUE TITLE
TypeError: can't access proprty "start" after following guide

### ISSUE THREAD

**Issue by jcousins-ticknovate** [uselessdev/datepicker#67]
Following the [installation and usage guide](https://github.com/uselessdev/datepicker#installation-and-usage), using the component throws the following error:

```sh
Uncaught TypeError: can't access property "start", value is undefined
    Calendar$1 uselessdev-datepicker.esm.js:294
    React 11
    workLoop scheduler.development.js:266
    flushWork scheduler.development.js:239
    performWorkUntilDeadline scheduler.development.js:533
    js scheduler.development.js:571
    js scheduler.development.js:633
    __require chunk-MB5FDF5C.js:12
    js index.js:6
    __require chunk-MB5FDF5C.js:12
    React 2
    __require chunk-MB5FDF5C.js:12
    js React
    __require chunk-MB5FDF5C.js:12
    js React
    __require chunk-MB5FDF5C.js:12
    <anonymous> react-dom_client:1
uselessdev-datepicker.esm.js:294:6
```

```sh
The above error occurred in the <Calendar$1> component:

Calendar$1@https://8fqih0.csb.app/node_modules/@uselessdev/datepicker/dist/uselessdev-datepicker.esm.js:326:18
EnvironmentProvider@https://8fqih0.csb.app/node_modules/@chakra-ui/react-env/dist/index.esm.js:119:62
ColorModeProvider@https://8fqih0.csb.app/node_modules/@chakra-ui/color-mode/dist/index.esm.js:183:15
ThemeProvider@https://8fqih0.csb.app/node_modules/@emotion/react/dist/emotion-element-b63ca7c6.cjs.dev.js:106:21
ThemeProvider@https://8fqih0.csb.app/node_modules/@chakra-ui/system/dist/index.esm.js:182:21
ChakraProvider@https://8fqih0.csb.app/node_modules/@chakra-ui/provider/dist/index.esm.js:20:177
ChakraProvider@https://8fqih0.csb.app/node_modules/@chakra-ui/react/dist/index.esm.js:110:24
App@https://8fqih0.csb.app/src/App.js:26:41

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries. 
    in Unknown index.js:27:25
```

This is replicated in the following codesandbox:

https://codesandbox.io/embed/uselessdev-datepicker-demo-8fqih0

Using this code:

```jsx
import { useState } from "react";
import { ChakraProvider } from "@chakra-ui/react";
import {
  Calendar,
  CalendarDefaultTheme,
  CalendarControls,
  CalendarPrevButton,
  CalendarNextButton,
  CalendarMonths,
  CalendarMonth,
  CalendarMonthName,
  CalendarWeek,
  CalendarDays
} from "@uselessdev/datepicker";

export default function App() {
  const [dates, setDates] = useState();

  const handleSelectDate = (values) => setDates(values);

  return (
    <>
      hello world
      <ChakraProvider theme={CalendarDefaultTheme}>
        <Calendar value={dates} onSelectDate={handleSelectDate}>
          <CalendarControls>
            <CalendarPrevButton />
            <CalendarNextButton />
          </CalendarControls>

          <CalendarMonths>
            <CalendarMonth>
              <CalendarMonthName />
              <CalendarWeek />
              <CalendarDays />
            </CalendarMonth>
          </CalendarMonths>
        </Calendar>
      </ChakraProvider>
    </>
  );
}
```

**Comment by uselessdev** [comment]
Hi @jcousins-ticknovate the problem is that your initial state must be an object:

`const [dates, setDates] = useState({});`

(yes the readme is missing this information) you can see a better example here: https://uselessdev-datepicker.netlify.app/?path=/docs/calendar--basic

**Comment by evolutionxbox** [comment]
Thanks. That closes #67
